### ISSUE TITLE
Spike: Isolate the web project from Xperience specifics completely

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#23]
### Expected result

Can we safely and efficiently decouple the front facing MVC project from Xperience implementation details?

IOW, how hard would it be to get rid of `using Kentico` statements in the MVC project?

### Additional context

Current areas of direct usage of Xperience code:

* service registration
* OWIN pipeline registration
* page builder & form builder components
* various helpers

### Resources

* URL

### Issue expands upon

This issue expands upon another issue(s): #1 

### Blockers

This issue is blocked by the following issue(s): #`issue number`
